pkg/scheduler/flavorassigner/flavorassigner_test.go

```diff
@@ -862,13 +860,16 @@ func TestAssignFlavors(t *testing.T) {
 			clusterQueueUsage: resources.FlavorResourceQuantities{
 				{Flavor: "one", Resource: corev1.ResourceCPU}: 9_000,
 			},
-			cohortResources: &cohortResources{
-				requestableResources: resources.FlavorResourceQuantities{
-					{Flavor: "one", Resource: corev1.ResourceCPU}: 100_000,
-				},
-				usage: resources.FlavorResourceQuantities{
-					{Flavor: "one", Resource: corev1.ResourceCPU}: 9_000,
-				},
+			secondaryClusterQueue: utiltesting.MakeClusterQueue("test-secondary-clusterqueue").
+				ResourceGroup(
+					utiltesting.MakeFlavorQuotas("one").
+						Resource(corev1.ResourceCPU, "10").
```

Comment:
98, to match old numbers?

I'm not even sure cohort is relevant for this test, but I'm in favor of not changing anything/matching the previous setup - even if it is strangely defined

Reply:
agree